server/chat-commands/info.js

```diff
@@ -2339,6 +2339,19 @@ const commands = {
 	},
 	pickrandomhelp: [`/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements.`],
 
+	'!shuffle': true,
+	shuffle(target, room, user) {
+		if (!target) return false;
+		if (!target || !target.includes(',')) return this.parse('/help shuffle');
+		const args = target.split(',');
+		if (args.length < 3) return this.parse(`/pick ${args}`);
```

Comment:
```suggestion
		if (args.length < 3) return this.parse(`/pick ${target}`);
```

if you left it as args it would parse as "/pick [option1, option2]" and show the brackets in the output